<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Insertt</class>
 <widget class="QDialog" name="Insertt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 219);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3" rowstretch="5,0,0" columnstretch="0,0">
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item row="1" column="1">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="Save">
        <property name="styleSheet">
         <string notr="true">QPushButton {
        background-color: #54e68a;
        color: white;
        border-radius: 10px;
        padding: 6px 12px;
        font-weight: bold;
    }
    QPushButton:hover {
        background-color: #45d17d;
	color:rgb(0,0,0);
    }
</string>
        </property>
        <property name="text">
         <string>SET GOAL</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancel">
        <property name="styleSheet">
         <string notr="true">QPushButton {
        background-color: #f56464;
        color: white;
        border-radius: 10px;
        padding: 6px 12px;
        font-weight: bold;
    }
    QPushButton:hover {
        background-color: #e25050;
	color:rgb(0,0,0)
    }</string>
        </property>
        <property name="text">
         <string>CANCEL</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <widget class="QFrame" name="frame">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>35</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <widget class="QLabel" name="label_pic">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>151</width>
          <height>221</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>goal2.jpg</pixmap>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">font: 600 14pt &quot;Segoe UI&quot;;
background-color: rgb(221, 193, 255);
radius:6px
</string>
     </property>
     <property name="title">
      <string>Vision</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="topMargin">
       <number>15</number>
      </property>
      <property name="bottomMargin">
       <number>19</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Name of your goal:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="goalNameEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Income required:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="incomeEdit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Downpayment:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="downpaymentEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Time limit:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="durationEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>Insertt</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>185</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Save</sender>
   <signal>clicked()</signal>
   <receiver>Insertt</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
